build

```
@@ -122,7 +122,8 @@ library() {
 			case "$arch" in # converts Alpine Linux arch strings to Docker arch strings
 				x86_64) echo -n "amd64";;
 				x86) echo -n "i386";;
-				armhf) echo -n "arm32v6";; # arm32v7 is not officially supported by Alpine, but arm32v6 should still work.
+				armhf) echo -n "arm32v6";;
+				armv7) echo -n "armv7";;
```

Sorry, by RHS, I meant Right-Hand-Side:

```suggestion
				armv7) echo -n "arm32v7";;
```

---

It's also worth noting that I'm not sure if this change is all that's required to add another architecture here -- I think there are more files that need to change for everything to work.

@ncopa @andyshinn any comment/suggestion?

I think it's also worth noting that 3.9 doesn't appear to have a 3.9 "minirootfs" artifact for `armv7`: http://dl-cdn.alpinelinux.org/alpine/v3.9/releases/armv7/

(Compare that to `armhf`: http://dl-cdn.alpinelinux.org/alpine/v3.9/releases/armhf/)

---

Yeah, I think it needs: https://github.com/oliv3r/aports/tree/armv7 to actually generate the images.
Anyway, most important part is to get the ball rolling right ;)